pkg/operator/ceph/controller/spec.go

```diff
@@ -75,21 +75,64 @@ var (
 	cronLogRotate = `
 CEPH_CLIENT_ID=%s
 PERIODICITY=%s
-LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
+LOG_ROTATE_CEPH_FOLDER=/etc/logrotate.d/ceph
+DAEMON_LOG_FILE=/var/log/ceph/
+LOG_MAX_SIZE=%s
+
+# size of log after removing any alphabet in the end. eg: for '1G' getting '1'
+LOG_MAX_SIZE_IN_DIGIT=$(echo "${LOG_MAX_SIZE}" | sed 's/.$//')
+# get last char of file size. eg: for '1G' getting 'G'
+GET_LAST_CHAR_OF_LOG_SIZE=$(echo $LOG_MAX_SIZE | rev | cut -c1)
+
+if [ "$GET_LAST_CHAR_OF_LOG_SIZE" == "G" ]; then
+	LOG_MAX_SIZE_IN_KB=$(( (LOG_MAX_SIZE_IN_DIGIT *1024*1024)))
+elif [ "$GET_LAST_CHAR_OF_LOG_SIZE" == "M" ]; then
+	LOG_MAX_SIZE_IN_KB=$(( (LOG_MAX_SIZE_IN_DIGIT *1024)))
+fi
+
+LAST_ROTATION_TIME_IN_SEC=$(echo $SECONDS)
 
 if [ -z "$PERIODICITY" ]; then
 	PERIODICITY=24h
 fi
 
+# remove last char from PERIODICITY eg. for 24h getting 24.
+PERIODICITY_IN_DIGIT=$(echo $PERIODICITY | awk '{print $1}' | sed 's/.$//')
```

Comment:
Do we assume the suffix is always hours? From the comment in cluster.yaml, seems like days are also supported.
```
periodicity: 24h # SUFFIX may be 'h' for hours or 'd' for days.
```